tests/yam/agama/boot_encrypted.pm

```diff
@@ -0,0 +1,44 @@
+## Copyright 2024 SUSE LLC
```

Comment:
is ok the header? could you compare with other? looks strange to me. Should be 2025.